### DISCUSSION TITLE
"Search" option on AIR SDK Reference (https://airsdk.dev/reference) ?

### DISCUSSION THREAD

**Discussion by 2jfw** [airsdk/Adobe-Runtime-Support#2672] [Q&A] [answered]
Is it planned to add a search option on the AIR SDK Reference on https://airsdk.dev/reference?
Old Adobe site had one (https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/index.html)

**Comment by ajwfrost** [comment, marked as answer]
I note the use of tense there... "Adobe site had one". It hasn't worked for a while :-)

Best we can offer at the moment is to use the "index" page and select the first letter of what you're looking for. The tools that generate all this don't have a 'search' option in them, but we can check to see if there are any variants of the scripts/templates that may be able to provide something like this, or whether that's an option to add in to a template somehow....

**Reply by 2jfw** [reply]
> I note the use of tense there... "Adobe site had one". It hasn't worked for a while :-)

Yeah, just realized that too, now 😅
Most people are likely coming from a search engine redirect anyways, so they already "found" what they were looking for ...

**Reply by marchbold** [reply]
Search has been on my list of things to look into for airsdk.dev, although was initially just going to be for the docs.

Should we have separate searches for the reference and docs?

Honestly would love to update the reference to be a bit more modern, taking the output from Harman / Adobe and converting it to something better but haven't even started looking at that yet :)

**Reply by ajwfrost** [reply]
@marchbold that would be awesome! I can probably share with you the XML files that are generated as part of the documentation process, they're then put through an XSLT process to turn them into the HTML/JS files...

> Should we have separate searches for the reference and docs?

How about a search textfield with a checkbox for 'search language reference only'? That's what Adobe had :-)
